src/components/app.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import axios from 'axios';
+import SearchForm from './search-form';
+import '../styles/app.scss';
+import '../styles/forecast-summary.scss';
+import '../styles/forecast-summaries.scss';
+import '../styles/search-form.scss';
+import '../styles/forecast-details.scss';
+
+const URL = 'https://mcr-codes-weather.herokuapp.com/forecast?city=';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchText: '',
+      searchError: '',
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        country: '',
+      },
+    };
+  }
+
+  componentDidMount() {
+    this.locationSearch('manchester');
+  }
+
+  handleForecastSelect = (date) => {
+    this.setState({
+      selectedDate: date,
+    });
+  };
+
+  handleInputChange = (event) => {
+    this.setState({ searchText: event.target.value });
+    if (event.target.value === '') {
+      this.setState({ searchError: '' });
+    }
+  };
+
+  handleKeyDown = (event) => {
+    if (event.key === 'Enter') {
+      this.setState(this.locationSearch(this.state.searchText));
+    }
+  };
+
+  locationSearch = (city) => {
+    axios.get(`${URL}${city}`)
+      .then((response) => {
+        this.setState({
+          forecasts: response.data.forecasts,
+          location: response.data.location,
+          searchError: '',
+        });
+      })
+      .catch(error => {
```

Comment:
This is a good general error message, but it may not alway be true. Check the type of information you might get when a request send back an error. You might be able to set your `searchError:` to something simple like `response.error.message`. If not you could use the information to decide which error message to display.